src/SourceBuild/content/eng/build.targets

```
@@ -0,0 +1,52 @@
+<Project>
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetValidArchiveItems" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="GetClosestOfficialSdk" />
+  <UsingTask AssemblyFile="$(SdkArchiveDiffTasksAssembly)" TaskName="FindArchiveDiffs" />
+
+  <Target Name="ReportSdkArchiveDiffs"
+          AfterTargets="Build"
+          DependsOnTargets="DetermineSourceBuiltSdkVersion"
+          Condition="'$(ShortStack)' != 'true' AND '$(PortableBuild)' == 'true'" >
```

nit: We use lowercase msbuild operators in this repo:

 ```suggestion
          Condition="'$(ShortStack)' != 'true' and '$(PortableBuild)' == 'true'" >
```